integration-test/src/test/java/org/sagebionetworks/IT970UserProfileController.java

```diff
@@ -130,94 +117,16 @@ public void testFavoriteCrud() throws Exception {
-		entity2.setName("bb");
-		entity2 = synapse.createEntity(entity2);
-		entitiesToDelete.add(entity2.getId());
-
-		acl = synapse.getACL(entity2.getId());
-		teamAccess = new ResourceAccess();
-		teamAccess.setAccessType(Collections.singleton(ACCESS_TYPE.READ));
-		teamAccess.setPrincipalId(Long.parseLong(teamToDelete));
-		acl.setResourceAccess(Sets.union(acl.getResourceAccess(), Collections.singleton(teamAccess)));
-		synapse.updateACL(acl);
-
+		// here we are not testing business logic, just making sure evewrything is "wired up" right:
+		
 		// retrieve my projects
-		PaginatedResults<ProjectHeader> projects = synapse.getMyProjects(ProjectListType.ALL, null, null, Integer.MAX_VALUE, 0);
-		assertEquals(2, projects.getTotalNumberOfResults());
-		assertEquals(2, projects.getResults().size());
-		List<ProjectHeader> alphabetical = Lists.newArrayList(projects.getResults());
-		Collections.sort(alphabetical, new Comparator<ProjectHeader>() {
-			@Override
-			public int compare(ProjectHeader o1, ProjectHeader o2) {
-				return o1.getName().compareTo(o2.getName());
-			}
-		});
-
-		PaginatedResults<ProjectHeader> projects5 = adminSynapse.getProjectsForTeam(Long.parseLong(teamToDelete),
-				ProjectListSortColumn.PROJECT_NAME, SortDirection.ASC, 100, 0);
-		// alphabetical
-		assertEquals(nullOutLastActivity(alphabetical), nullOutLastActivity(projects5.getResults()));
-
-		// change order
-		folder.setName("folder1-renamed");
-		EntityBundleCreate ebc = new EntityBundleCreate();
-		ebc.setEntity(folder);
-		synapse.updateEntityBundleV2(folder.getId(), ebc);
-
-		TimeUtils.waitFor(MAX_WAIT_MS, 1000, Lists.reverse(projects.getResults()), new Predicate<List<ProjectHeader>>() {
-			@Override
-			public boolean apply(List<ProjectHeader> expected) {
-				try {
-					PaginatedResults<ProjectHeader> projects = synapse.getMyProjects(ProjectListType.ALL, null, null,
-							Integer.MAX_VALUE, 0);
-					return expected.equals(projects.getResults());
-				} catch (SynapseException e) {
-					throw new RuntimeException(e.getMessage(), e);
-				}
-			}
-		});
-
-		PaginatedResults<ProjectHeader> projects3 = synapse.getMyProjects(ProjectListType.ALL, null, null, Integer.MAX_VALUE, 0);
-		assertEquals(nullOutLastActivity(Lists.reverse(projects.getResults())), nullOutLastActivity(projects3.getResults()));
-
-		PaginatedResults<ProjectHeader> projects4 = adminSynapse.getProjectsFromUser(userToDelete, null, null, 100, 0);
-		assertEquals(nullOutLastActivity(Lists.reverse(projects.getResults())), nullOutLastActivity(projects4.getResults()));
-
-		projects5 = adminSynapse.getProjectsForTeam(Long.parseLong(teamToDelete), ProjectListSortColumn.PROJECT_NAME, SortDirection.ASC, 100,
-				0);
-		// still alphabetical
-		assertEquals(nullOutLastActivity(alphabetical), nullOutLastActivity(projects5.getResults()));
+		ProjectHeaderList projects = synapse.getMyProjects(ProjectListType.ALL, null, null, null);
+		// retrieve someone else's projects
```

Comment:
comment applies to line 128.  this comment should be something like "retrieve my second page of projects"